python/Ganga/GPIDev/Lib/File/GoogleFile.py

```diff
@@ -48,11 +56,11 @@ class GoogleFile(IGangaFile):
                       'localDir': SimpleItem(defvalue="", copyable=1,
                                              doc='local dir where the file is stored, used from get and put methods'),
                       'subfiles': ComponentItem(category='gangafiles', defvalue=[], hidden=1,
-                                                typelist=['Ganga.GPIDev.Lib.File.LCGSEFile'], sequence=1, copyable=0,
+                                                typelist=[GoogleFileStub], sequence=1, copyable=0,
```

Comment:
For now is there agreement to remove these typelist from the IGanagFile.sufiles objects?